enterprise/admin/__init__.py

```diff
@@ -207,8 +207,8 @@ class EnterpriseCustomerAdmin(DjangoObjectActions, SimpleHistoryAdmin):
                        'enable_generation_of_api_credentials')
         }),
         ('Recommended default settings for all enterprise customers', {
-            'fields': ('site', 'customer_type', 'enable_learner_portal',
-                       'enable_integrated_customer_learner_portal_search',
+            'fields': ('site', 'customer_type', 'disable_expiry_messaging_for_learner_credit',
```

Comment:
nit/suggestion: given this field likely is not one that will be frequently used for most customers, I wonder if we should de-prioritize it within this grouping. For example, `enable_learner_portal`, etc. are more critical customer config settings that I feel should be displayed first in this section for usability/discoverability of the more common fields.

I might suggest perhaps grouping the new field alongside the other learner-credit related settings (see screenshot), or towards the last item in this section.:

<img width="882" alt="image" src="https://github.com/openedx/edx-enterprise/assets/2828721/323205d7-cd62-44c0-afbf-338cc70d8726">

Reply:
I went ahead and moved it right below the enablement of learner credit.
![Screenshot 2024-05-16 at 12 23 43 PM](https://github.com/openedx/edx-enterprise/assets/82611798/baaecf92-9a4e-43a6-8714-c574429396f7)